app/views/tasks/_form.html.erb

```diff
@@ -0,0 +1,20 @@
+<!--each one is a statement not a hidden visible ** just make it a list-->
+
+    <h2>TASKS</h2>
+      <!-- <p><%#= link_to "Edit", edit_task_path(task.id) %></p> -->
+      <!--<p><%#= link_to "Delete", delete_task_path(task.id) %></p> -->
+    <!-- <ol> -->
+      <% @tasks.each_with_index do |task, i|  %>
+        <section>
+          <h3><%= "#{i + 1}.    " %><%= link_to task.title, task_path(task.id), { :class => (task.complete) ? "completed" : ""} %></h3>
+
+          <% if task.complete == true %>
+            <p><%= link_to "Unmark Complete", task_path(task.id, {:task => {:complete => false}, :refresh => true}), method: :patch %></p>
+            <% else %>
+             <p><%= link_to "Mark Complete", task_path(task.id, {:task => {:complete => true, :completed_date => DateTime.now}, :refresh => true}), method: :patch %></p>
+          <% end %>
+
+          <p><%= link_to "Edit", task_path(task.id) %></p>
```

Comment:
In order for this link to go to the edit form for a given task, we would need to use the `edit_task_path` route helper method instead of the `task_path` helper, which would take us to the show page.